### PR TITLE
Proxies and egress_blocking

### PR DESCRIPTION
**Pull request by Zempashi**
Fix: #123

### CODE REVIEW

**Comment by Zempashi** [comment]
@bert-e wait

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ansible.cfg
 
 *.pyc
-/*.retry
+*.retry
 /playbooks/*.retry
```
Comment (ballot-scality) [inline .gitignore]: Useless now

File: contrib/playbooks/roles/squid_proxy/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+- name: 'install squid package'
+  package:
+    name: squid
+    state: present
+  register: squid_pkg_just_installed
+
+#- name: 'template squid configuration'
```
Comment (ballot-scality) [inline contrib/playbooks/roles/squid_proxy/tasks/main.yml]: Comment ?

File: contrib/terraform/openstack/common_nodes.tf [hunk 3]
```diff
@@ -24,6 +24,11 @@ variable "ssh_user" {
   default = "centos"
 }
 
+variable "local_link_config_address_ip_v4" {
+  type    = "string"
+  default = "169.254.169.254"
```
Comment (ballot-scality) [inline contrib/terraform/openstack/common_nodes.tf:31]: A little comment on what is this IP

File: contrib/terraform/openstack/etcd.tf [hunk 4]
```diff
@@ -25,30 +26,54 @@ resource "openstack_compute_instance_v2" "etcd" {
...
+# egress when default egress rules are removed
+resource "openstack_networking_secgroup_rule_v2" "etcd_to_etcd_egress" {
+  direction         = "egress"
+  ethertype         = "IPv4"
+  remote_group_id   = "${openstack_networking_secgroup_v2.etcd.id}"
+  security_group_id = "${openstack_networking_secgroup_v2.etcd.id}"
+  count             = "${local.egress_blocked ? 1 : 0}"
+}
+
+#resource "openstack_networking_secgroup_rule_v2" "etcd_to_masters_egress" {
```
Comment (ballot-scality) [inline contrib/terraform/openstack/etcd.tf]: Comments

File: contrib/terraform/openstack/inventory.tf [hunk 5]
```diff
@@ -14,16 +14,28 @@ locals {
         slice(local.etcd_name, 0, local.masters_on_etcd)
   )}"]
 
+  proxies_name = ["${openstack_compute_instance_v2.proxies.*.name}"]
+
+  proxy_url = "${coalesce(
```
Comment (ballot-scality) [inline contrib/terraform/openstack/inventory.tf:21]: A small commentary ? :)

Comment (Zempashi) [reply contrib/terraform/openstack/inventory.tf:21]: Version a bit simpler, and a comment

**Comment by Zempashi** [comment]
@bert-e reset

**Comment by Zempashi** [comment]
@bert-e reset

**Review by ballot-scality — APPROVED** [review]
LGTM

**Comment by Zempashi** [comment]
@bert-e approve

**Pull request merged**